saleor/graphql/order/mutations/discount_order.py

```diff
@@ -200,7 +185,7 @@ def perform_mutation(cls, root, info, **data):
         order_discount.value_type = value_type
         order_discount.save()
 
-        cls.recalculate_order(order)
+        fetch_order_prices_if_expired(order, info.context.plugins, force_update=True)
```

Comment:
I think we need to take order webhook serialization into account (e.g. order_updated). This means that the order serializer should actually use the calculations module instead of relying on the model attributes

Reply:
Yes